apps/web/src/components/auth/SignUpForm.tsx

```diff
@@ -216,7 +216,7 @@ export function SignUpForm({ token, email }: Props) {
           submit
           data-test-id="submitButton"
         >
-          Sign Up {token ? '& Accept Invite' : null}
+          Sign Up {invitationToken ? '& Accept Invite' : null}
```

Comment:
Same as in the LoginForm, for me it makes more sense `Accept Invite & Sign Up` as it is the order of the actions happening. Maybe I am petty with the wording though.

Reply:
yeah, the action flow depends on the user's choice for signup: GitHub button or form button.
For GH it's: sign-up & accept
For form it's: accept & sign up

Reply:
Oh yeah, you are right. Even more complicated. 😓